src/commands/tenmans.ts

```diff
@@ -214,6 +273,21 @@ const createEmbed = (time) =>
     .setTimestamp()
     .setFooter("Last Updated");
 
+const createVoteEmbed = (votesStillNeeded: number, time, closingTime: Date) =>
```

Comment:
Why not have a vote/withdraw vote button like in queues? The less slash commands the better IMO

Reply:
I'll spend some time adding this.

Reply:
After looking at how this might be implemented, my thoughts are that the refactoring required is too much effort relative to what will be lost upon a refactor that supports multiple queues. We'd need to further abstract the QueueAction which I'd rather do once we know what multiple queues looks like on the implementation side.